canonical/ght
# Added regions

## Done 
New regions are added

## QA
- Run `yarn dev -i`
- Select job, job post
- Select "emea" region
- Verify job posts are created at Valencia, Murcia, Sevilla, Granada, Salamanca, Coimbra, Aveiro, Porto, Torino / Turin, Pisa, Toulouse, Grenoble, Bordeaux, Metz, Lille, Rennes, Nice.

Fixes #135
